tools/downloader/requirements-pytorch.in

```diff
@@ -3,3 +3,4 @@ scipy           # via torchvision
 torch>=1.8
 torchvision
 yacs
+addict          # via text-recognition-resnet-fc
```

Comment:
@akorobeinikov @morkovka1337 can this package be downloaded and installed (unzipped) from pypi during model downloader work?

Reply:
Now package is installed by model downloader.